<robot name="AGV">
    <link name="base_link">
        <visual>
            <origin rpy="0 0 0" xyz="0 0 0.025"/>
            <geometry>
                <box size="0.82 .49 .195"/>
            </geometry>
            <material name="gray">
                <color rgba="0 0 0 0.6"/>
            </material>
        </visual>
    </link>

    <link name="wheel_left">
        <visual>
	    <origin rpy="1.57079633 0 0" xyz="0 0 0"/>
            <geometry>
                <cylinder length="0.04" radius="0.1"/>
            </geometry>
            <material name="black">
                <color rgba="1 0 0 1"/>
            </material>
        </visual>
    </link>

    <joint name="base_to_wheel_left" type="continuous">
        <parent link="base_link"/>
        <child link="wheel_left"/>
        <origin xyz="0 -0.205 0.01" rpy="0 0 0"/>
        <axis xyz="0 1 0" rpy="0 0 0"/>
    </joint>

    <link name="wheel_right">
        <visual>
            <geometry>
                <cylinder length="0.04" radius="0.1"/>
            </geometry>
            <origin rpy="1.57079633 0 0" xyz="0 0 0"/>
            <material name="black"/>
        </visual>
    </link>

    <joint name="base_to_wheel_right" type="continuous">
        <parent link="base_link"/>
        <child link="wheel_right"/>     
        <origin xyz="0 0.205 0.01" rpy="0 0 0"/>
        <axis xyz="0 1 0" rpy="0 0 0"/>
    </joint>
  </robot>
